.github/workflows/pkgci_regression_test.yml

```diff
@@ -91,6 +91,13 @@ jobs:
       #       -rA -s -m "plat_host_cpu and presubmit" \
       #       experimental/regression_suite
 
+      - name: "Set IREE_TEST_FILES variable" 
+        run: |
+          case "${{ matrix.name }}" in
+            "amdgpu_rocm_mi300_gfx942") IREE_TEST_FILES="/shark-cache/data/iree-regression-cache" >> $GITHUB_ENV;;
+            *) echo "No cache directory assigned for ${{ matrix.name }}" ;;
+          esac
```

Comment:
@ScottTodd This doesn't work unless I conditionally set IREE_TEST_FILES in the specific run clause 
https://github.com/iree-org/iree/actions/runs/14045768606/job/39326526495#step:7:207

Reply:
You need to write to the file, e.g. using `echo >>`. Test in a smaller workflow in a test repository / fork if you need to.